package.json

```diff
@@ -113,4 +113,4 @@
       "path": "node_modules/cz-conventional-changelog"
     }
   }
-}
+}
```

Comment:
There should be a new line here as it was before.

Reply:
My IDE always deletes the line :-/

Reply:
You can configure it to always add it, not remove. :)